prototests/test_splineBasis.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from neurostatslib.basis import BSplineBasis, Cyclic_BSplineBasis
+import numpy as np
+
+np.random.seed(10**5)
```

Comment:
I like to feed the seed into the test function like

```python
@pytest.mark.parametrize('seed', [111, 222, 333])
def test_me(seed):
    rng = np.random.RandomState(seed)
    random_numbers = rng.randn(100)
```

Reply:
I changed completely how the test is set up, let me know if you like this way of structuring it more